sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -242,16 +705,15 @@ To learn more about other logging mechanisms see [Diagnostics Samples][logging].
 
 Samples showing how to use the Cognitive Services Form Recognizer library are available in this GitHub repository. Samples are provided for each main functional area:
 
-- [Recognize form content][recognize_content]
-- [Recognize custom forms][recognize_custom_forms]
-- [Recognize receipts][recognize_receipts]
-- [Recognize business cards][recognize_business_cards]
-- [Recognize invoices][recognize_invoices]
-- [Recognize identity documents][recognize_identity_documents]
-- [Train a model][train_a_model]
-- [Manage custom models][manage_custom_models]
+- [Extract the layout of a document][extract_layout]
```

Comment:
not sure what the right .NET thing to do is here, but consider linking to samples README rather than providing individual links here so people can choose the samples that match the API version they are targeting

Reply:
We want to encourage customers to use the new API version, so I think there's value in keeping it as it is. We have a minor disclaimer at the end of the samples:
> Note that these samples use SDK `V4.0.0-beta.X`. For lower versions of the SDK, please see [Form Recognizer Samples for V3.1.X][formrecov3_samples].

I added it to the start of the samples section as well to make sure it's more discoverable.